tests/test_dump_basic.py

```diff
@@ -87,17 +87,16 @@ async def test_dump_fuzz_01(self):
         dbname = self.get_database_name()
         restored_dbname = f'{dbname}_restored'
 
-        with tempfile.TemporaryDirectory() as f:
```

Comment:
This test was failing and it appears that it was timing-related. I reordered the create/dump steps to make it more reliable.